Examples/Swift/SwitchStylesExample.swift

```diff
@@ -0,0 +1,48 @@
+import UIKit
+import Mapbox
+
+@objc(SwitchStylesExample_Swift)
+
+class SwitchStylesExample: UIViewController, MGLMapViewDelegate {
+    
+    let mapView = MGLMapView()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView.frame = CGRect(x: 0, y: 0, width: view.bounds.width, height: view.bounds.height)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.styleURL = MGLStyle.streetsStyleURL()
+        mapView.centerCoordinate = CLLocationCoordinate2D(latitude: 28.10, longitude: -81.76)
+        mapView.setZoomLevel(5.4, animated: false)
+        view.addSubview(mapView)
+        mapView.delegate = self
+        
+        let styleToggle = UISegmentedControl(items: ["Dark", "Streets", "Light"])
+        styleToggle.translatesAutoresizingMaskIntoConstraints = false
+        styleToggle.selectedSegmentIndex = 1
+        view.insertSubview(styleToggle, aboveSubview: mapView)
+        styleToggle.addTarget(self, action: #selector(changeStyle(sender:)), for: .valueChanged)
+        
+        if #available(iOS 9.0, *) {
+            styleToggle.bottomAnchor.constraint(equalTo: mapView.logoView.topAnchor, constant: -20).isActive = true
+            styleToggle.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+            styleToggle.widthAnchor.constraint(equalTo: view.widthAnchor, constant: -80).isActive = true
+        } else {
+            // Fallback on earlier versions
```

Comment:
We should setup up constraints using the legacy API here _or_ bump the deployment target to iOS 9 and remove this fallback. I'd suggest the latter since iOS 8 and below seems to be about 0.6%.

Reply:
Since the Maps SDK has a minimum deployment target of iOS 8, we probably shouldn't diverge the iOS examples from that.